content/guides/core-concepts/introduction-to-cypress.md

```diff
@@ -791,9 +815,19 @@ commands that were enqueued using the `cy.*` command chains.
 
 #### Let's take another look at an example
 
+:::visit-mount-test-example
```

Comment:
For consistency...

```js
it('hides the thing when it is clicked', () => {
  cy.visit('/my/resource/path') // 1

  cy.get('.hides-when-clicked') // 2
    .should('be.visible')       // 3
    .click()                    // 4
    .should('not.be.visible')   // 5
})
```

```js
it('hides the thing when it is clicked', () => {
  cy.mount(<MyComponent />)     // 1

  cy.get('.hides-when-clicked') // 2
    .should('be.visible')       // 3
    .click()                    // 4
    .should('not.be.visible')   // 5
})
```

And:

> The test above would cause an execution in this order:
> 
> 1. Visit the URL (or mount the component).
> 2. Find an element by its selector.
> 3. Assert that the element is visible.
> 4. Perform a click action on that element.
> 5. Assert that the element is no longer visible.

And then after that:

> To illustrate this, let's revisit that list of actions and expose some of the hidden ✨ **magic** ✨ Cypress does for us at each step:
> 
> 1. Visit the URL ✨ **and wait for the page load event to fire after all external resources have loaded**✨ (or mount the component ✨ **and wait for the component to finish mounting**✨)
> 2. Find an element by its selector ✨ **and retry until it is found in the DOM** ✨
> 3. Assert that the element is visible ✨ **and retry until the assertion passes** ✨
> 4. Perform a click action on that element ✨ **after we wait for the element to reach an actionable state** ✨
> 5. Assert that the element is no longer visible ✨ **and retry until the assertion passes** ✨